_posts/2024-04-14-blog.md

```diff
@@ -0,0 +1,334 @@
+    <div class="text">Spread of replicates with 90% of genotypes replaced by missing data (~ 60.000 SNPs left).</div>
+  </div>
+
+  <div class="mySlides slide2 fade">
+    <div class="numbertext">2 / 3</div>
+    <img src="{{ "/assets/media/event_images/2024-04-14-blog/WestEurasia.m0.97.png" | relative_url }}" style="width:90%">
+   <br>
+    <div class="text">Spread of replicates with 97% of genotypes replaced by missing data (~ 18.000 SNPs left).</div>
+  </div>
+
+  <div class="mySlides slide2 fade">
+    <div class="numbertext">3 / 3</div>
+    <img src="{{ "/assets/media/event_images/2024-04-14-blog/WestEurasia.m0.99.png" | relative_url }}" style="width:90%">
+   <br>
+    <div class="text">Spread of replicates with 99% of genotypes replaced by missing data (~ 6.000 SNPs left).</div>
+  </div>
+
+  <!-- Next and previous buttons -->
+  <a class="prev" onclick="plusSlides(-1, 2)">&#10094;</a>
+  <a class="next" onclick="plusSlides(1, 2)">&#10095;</a>
+</div>
+<br>
+
+## Conclusions
+
+In literature, the chosen cutoffs are both sensible and (strictly speaking) arbitrary, commonly ranging between 10.000 and 20.000 covered SNPs.
+Any individuals with fewer than that number of SNPs are often excluded from downstream analyses.
+Looking at the results above, you can see why:
+
+ - The degree of variation in placement of the replicates at high missingness rates can make inferences of local ancestry based on PCA erroneous.
+    - e.g. with sparse enough data (99% missing), some replicates of the Sicilian individual 'TP08' are projected closer to present-day Greeks than the other Sicilians in the dataset. Indeed, with 18.000 SNPs (97% missing data), the spread of replicates is roughly halved (visually).
+
+These cutoffs are not absolute, however. There are still things we can infer about individuals with a low number of covered SNPs:
+
+ - While the exact projection of low-coverage individuals in the Worldwide PCA is also affected, the spread of replicates shows that it is still perfectly possible to assess if a studied individual is genetically more closely related to present-day African, South Asian or Native American populations from the PCA results, even from only a few thousand SNPs.
+   - While not shown here, it should be noted that differentiating between Asian and Native American populations (whose positions are very similar in the Worldwide PCA plot above), will be possible when considering additional PCs. 
+
+An additional thing to note about the results from this analysis (especially when looking at the West Eurasian PCA) is that the spread of replicates around the 'Loschbour' hunter-gatherer is not visibly different for that around other candidate individuals. 
+This suggests that damage artefacts, the pseudohaploid genotyping process, as well as the fact that this individual is not in the 'poplist' of the PCA (and hence its variation is not used in the calculation of the PCs), do not seem to exacerbate the spread of replicates.
```

Comment:
I think referring to it was projected (unlike the others, if I understand) would make more sense. You mention projection in the exeception above but not here, and then introduce the 'poplist' here which is otherwise not mentioned